tests/functests/CMakeLists.txt

```diff
@@ -35,7 +35,14 @@ if (EXISTS ${CMAKE_SOURCE_DIR}/lib/modules/exp/tests)
         ${CMAKE_SOURCE_DIR}/lib/modules/exp/tests/functests/ECSConfigCacheFuncTests.cpp
         )
     if (EXISTS ${CMAKE_SOURCE_DIR}/lib/modules/exp/tests/functests/test.json)
-      file(COPY_FILE ${CMAKE_SOURCE_DIR}/lib/modules/exp/tests/functests/test.json ${CMAKE_BINARY_DIR}/test.json)
+      if(${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.19")
```

Comment:
It seems `COPY_FILE` was added in `3.21`?

https://cmake.org/cmake/help/latest/command/file.html

Reply:
Thanks!. Fixed